web/elm/src/Build/StepTree/StepTree.elm

```diff
@@ -1074,16 +1074,22 @@ viewStepHeaderLabel : StepHeaderType -> StepID -> Html Message
 viewStepHeaderLabel headerType stepID =
     let
         eventHandlers =
-            if headerType == StepHeaderGet True then
-                [ onMouseLeave <| Hover Nothing
-                , onMouseEnter <| Hover <| Just <| FirstOccurrenceGetStepLabel stepID
-                ]
+            case headerType of
+                StepHeaderGet True ->
+                    [ onMouseLeave <| Hover Nothing
+                    , onMouseEnter <| Hover <| Just <| ChangedStepLabel stepID "new version"
+                    ]
 
-            else
-                []
+                StepHeaderSetPipeline True ->
+                    [ onMouseLeave <| Hover Nothing
+                    , onMouseEnter <| Hover <| Just <| ChangedStepLabel stepID "pipeline config changed"
+                    ]
+
+                _ ->
+                    []
     in
     Html.div
-        (id (toHtmlID <| FirstOccurrenceGetStepLabel stepID)
+        (id (toHtmlID <| ChangedStepLabel stepID "")
```

Comment:
`""` feels a bit weird but I think the alternative would have to be keeping the `FirstOccurrenceGetStepLabel` and creating a new `ChangedSetPipelineLabel` (in `DomID`) but in almost every situation this is just going to result in another branch in a `case` statement with almost identical contents.